oci/private/tarball.bzl

```diff
@@ -14,7 +14,7 @@ and then run it in a container like so:
 
 ```
 bazel build //path/to:tarball
```

Comment:
can remove (every one of these example command edits should be two lines replaced by one, right?)